### PR TITLE
Add reference to ant-contrib to README, path reference to build.xml and guard to build.sh

### PR DESCRIPTION
**Pull request by theseyi**
This should help for users who don't have ant-contrib installed and don't know what parameters are needed in the build configuration. Let  me know if this works or otherwise?
I had issues cloning the repo and running because I didn't have ant-contrib and wasn't too familiar with ant

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,6 +34,8 @@ I've modified `vendor/libphonenumber/javascript/build.xml` and turned that into
 
 The `./build.sh` command uses those two pieces together in order to generate a new version of libphonenumber.js in the `/dist` folder.
 
+Before you run the build script, ensure you have [ant-contrib](http://ant-contrib.sourceforge.net/) installed, then replace `/PATH/TO/ANT-CONTRIB.JAR` with the actual path to your .jar file.
```
Comment (nathanhammond) [inline README.md]:
```diff
- then replace `/PATH/TO/ANT-CONTRIB.JAR` with the actual path to your .jar file
+ and present on your system's PATH.
```

**Comment by nathanhammond** [comment]
I don't want to manually specify the `PATH` and instead want users to add it to their system `PATH` so that it works out of the box. That removes the need to the modifications to `build.sh` and `build.xml`.

**Comment by theseyi** [comment]
That makes sense, are there any other configurations the user would have to do to allow `parallel="true"` generate the proper filename for each `dist/libphonenumber_${regioncode}.js`?

**Comment by nathanhammond** [comment]
Thanks for the PR! My apologies for not being familiar enough with all of this to be able to figure out the parallel story.

**Pull request merged**